hoomd/md/pair/pair.py

```diff
@@ -293,6 +293,63 @@ def __init__(self, nlist, default_r_cut=None, default_r_on=0., mode='none'):
         super().__init__(nlist, default_r_cut, default_r_on, mode)
 
 
+class ExpandedGaussian(Pair):
+    r"""Expanded Gaussian pair force.
+
+    Args:
+        nlist (hoomd.md.nlist.NeighborList): Neighbor list.
+        default_r_cut (float): Default cutoff radius :math:`[\mathrm{length}]`.
+        default_r_on (float): Default turn-on radius :math:`[\mathrm{length}]`.
+        mode (str): Energy shifting/smoothing mode.
+
+    `ExpandedGaussian` computes the Gaussian pair force should on every particle
+    in the simulation state:
```

Comment:
I'm not sure what this sentence is supposed to be saying. Is the word should supposed to be shifted?